src/control/server/storage/config.go

```diff
@@ -749,13 +749,21 @@ func (ap *AccelProps) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+// SpdkRpcServer struct describes settings for an optional SPDK JSON-RPC server instance that can
+// run in the engine process.
+type SpdkRpcServer struct {
+	Enable   bool   `yaml:"enable,omitempty" json:"enable"`
+	SockAddr string `yaml:"sock_addr,omitempty" json:"sock_addr"`
+}
+
 type Config struct {
-	Tiers            TierConfigs `yaml:"storage" cmdLongFlag:"--storage_tiers,nonzero" cmdShortFlag:"-T,nonzero"`
-	ConfigOutputPath string      `yaml:"-" cmdLongFlag:"--nvme" cmdShortFlag:"-n"`
-	VosEnv           string      `yaml:"-" cmdEnv:"VOS_BDEV_CLASS"`
-	EnableHotplug    bool        `yaml:"-"`
-	NumaNodeIndex    uint        `yaml:"-"`
-	AccelProps       AccelProps  `yaml:"acceleration,omitempty"`
+	Tiers            TierConfigs   `yaml:"storage" cmdLongFlag:"--storage_tiers,nonzero" cmdShortFlag:"-T,nonzero"`
+	ConfigOutputPath string        `yaml:"-" cmdLongFlag:"--nvme" cmdShortFlag:"-n"`
+	VosEnv           string        `yaml:"-" cmdEnv:"VOS_BDEV_CLASS"`
+	EnableHotplug    bool          `yaml:"-"`
+	NumaNodeIndex    uint          `yaml:"-"`
+	AccelProps       AccelProps    `yaml:"acceleration,omitempty"`
+	SpdkRpcSrvProps  SpdkRpcServer `yaml:"spdk_rpc_server,omitempty"`
```

Comment:
From what I understand the file `utils/config/daos_server.yml` should also be updated ?

Reply:
this option is for test purposes only and shouldn't be used in production, that's why I haven't updated the user documentation in `daos_server.yml`

Reply:
OK, I missed this